src/web/client/common/constants.ts

```diff
@@ -103,7 +103,8 @@ export enum queryParameters {
     WEBSITE_PREVIEW_URL = "websitepreviewurl",
     ENTITY = "entity",
     ENTITY_ID = "entityid",
-    REFERRER_SOURCE = "referrersource"
+    REFERRER_SOURCE = "referrersource",
+    ORG_GEO = 'orgGeo'
```

Comment:
lets leave a comment here that "geo" above means user geo - we shouldnt rename that param but lets leave a clarifying comment.